ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-840] Add unemployed partner attributes

## Description of change
Adds attributes required for unemployed partner journey

## Link to relevant ticket
[CRIMAPP-840](https://dsdmoj.atlassian.net/browse/CRIMAPP-840)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature

Could test an application all the way to resubmission to check it hasn't broken anything


[CRIMAPP-840]: https://dsdmoj.atlassian.net/browse/CRIMAPP-840?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
